rust/tvm/src/ir/expr.rs

```diff
@@ -94,9 +94,9 @@ external! {
     fn _as_text(object: ObjectRef, show_meta_data: i32, annotate: runtime::Function) -> TString;
 }
 
-pub fn as_text<T: IsObjectRef>(object: T) -> String {
+pub fn as_text<T: IsObjectRef>(object: T, show_meta_data: i32) -> String {
```

Comment:
1. Does the new argument break existing callsites of `as_text`?  Since we can't add default parameters to preserve backwards compatibility, I'm wondering if this should be updated to use the builder pattern in the future.
2. Should the argument be `show_meta_data: bool` instead of `show_meta_data: i32`?  The FFI cannot currently distinguish between the two, but exposing it as `i32` in the Rust bindings seems odd.

Reply:
> Does the new argument break existing callsites of `as_text`? Since we can't add default parameters to preserve backwards compatibility, I'm wondering if this should be updated to use the builder pattern in the future.

You mean `object.with_metadata().as_text()`? Unfortunately I am not sure wether backwards compatibility in this case is important enough, to warrant the increased API complexity.